src/store/actions/setting/index.ts

```diff
@@ -0,0 +1,46 @@
+import * as settingsConstants from './constants';
+
+export const fetchApplianceCategory = (payload): Action => ({
+  type: settingsConstants.FETCH_APPLIANCE_CATEGORY_REQUEST,
+  payload,
+});
+
+export const fetchParameters = (payload): Action => ({
+  type: settingsConstants.FETCH_PARAMETERS_REQUEST,
+  payload,
```

Comment:
Destructuring here would be nice as well in order not to  repeat settingsConstants

Reply:
Nice one.